FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -357,6 +360,12 @@ private async Task WaitRunningJobComplete(IProgress<string> progress, Cancellati
                                 _result.ProcessedResourceCounts.ConcatDictionaryCount(processingJobResult.ProcessedCount);
                             _result.SkippedResourceCounts =
                                 _result.SkippedResourceCounts.ConcatDictionaryCount(processingJobResult.SkippedCount);
+                            _result.ProcessedCountInTotal += processingJobResult.ProcessedCountInTotal;
+                            _result.ProcessedDataSizeInTotal += processingJobResult.ProcessedDataSizeInTotal;
+
+                            // log metrics
+                            _metricsLogger.LogSuccessfulResourceCountMetric(_result.ProcessedCountInTotal);
```

Comment:
what log here is the accumulated value instead of the incremental value, is it by design?

Reply:
Fixed it.